internal/target/query.go

```diff
@@ -61,5 +61,272 @@ select public_id, project_id from target
 
 	estimateCountTargets = `
 select sum(reltuples::bigint) as estimate from pg_class where oid in ('target_tcp'::regclass, 'target_ssh'::regclass)
+`
+
+	listTargetsTemplate = `
+with targets as (
+  select public_id
+  from target
+  -- search condition for applying permissions is constructed
+  where %s
+  order by create_time desc, public_id asc
+  limit %d
+),
+tcp_targets as (
+  select *
+  from target_tcp
+  where public_id in (select public_id from targets)
+),
+ssh_targets as (
+  select *
+  from target_ssh
+  where public_id in (select public_id from targets)
+),
+all_targets as (
+  select
+    public_id,
+    project_id,
+    name,
+    description,
+    default_port,
+    session_max_seconds,
+    session_connection_limit,
+    version,
+    create_time,
+    update_time,
+    worker_filter,
+    egress_worker_filter,
+    ingress_worker_filter,
+    default_client_port,
+    null as storage_bucket_id,
+    false as enable_session_recording,
+    'tcp' as type
+  from tcp_targets
+  union
+  select
+    public_id,
+    project_id,
+    name,
+    description,
+    default_port,
+    session_max_seconds,
+    session_connection_limit,
+    version,
+    create_time,
+    update_time,
+    worker_filter,
+    egress_worker_filter,
+    ingress_worker_filter,
+    default_client_port,
+    storage_bucket_id,
+    enable_session_recording,
+    'ssh' as type
+  from ssh_targets
+)
+select * 
+  from all_targets
+order by create_time desc, public_id asc;
```

Comment:
```suggestion
with targets as (
    select public_id
      from target
     -- search condition for applying permissions is constructed
     where %s
  order by create_time desc, public_id asc
     limit %d
),
tcp_targets as (
  select *
    from target_tcp
   where public_id in (select public_id from targets)
),
ssh_targets as (
  select *
    from target_ssh
   where public_id in (select public_id from targets)
),
all_targets as (
  select public_id,
         project_id,
         name,
         description,
         default_port,
         session_max_seconds,
         session_connection_limit,
         version,
         create_time,
         update_time,
         worker_filter,
         egress_worker_filter,
         ingress_worker_filter,
         default_client_port,
         null as storage_bucket_id,
         false as enable_session_recording,
         'tcp' as type
    from tcp_targets
   union
  select public_id,
         project_id,
         name,
         description,
         default_port,
         session_max_seconds,
         session_connection_limit,
         version,
         create_time,
         update_time,
         worker_filter,
         egress_worker_filter,
         ingress_worker_filter,
         default_client_port,
         storage_bucket_id,
         enable_session_recording,
         'ssh' as type
    from ssh_targets
)
  select *
    from all_targets
order by create_time desc, public_id asc;
```

Can you update the formatting of the queries like above. This creates a "river" so that keywords all line up and helps with readability of the queries.